cpp/ql/src/semmle/code/cpp/Variable.qll

```diff
@@ -286,8 +319,16 @@ class ParameterDeclarationEntry extends VariableDeclarationEntry {
 
 /**
  * A C/C++ variable with block scope [N4140 3.3.3]. In other words, a local
- * variable or a function parameter. Local variables can be static; use the
- * `isStatic` member predicate to detect those.
+ * variable or a function parameter. For example, the variables `a` and `b` in
+ * the following code:
+ * ```
+ * void myFunction(int a) {
+ *   int b;
+ * }
+ * ```
+ * 
+ * Local variables can be static; use the `isStatic` member predicate to
+ * detect those.
  */
```

Comment:
And here we could perhaps give an example of a static local variable:
```
void MyFunction(int a) {
  static int b;
}
```

Reply:
Good idea.  Added.